contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+== Mecánica de la contribución
+
+¿Estás preparado para empezar a contribuir a los proyectos/repositorios de otros equipos?
+¿Quieres reducir tus bloqueos colaborando en vez de gestionar?
```

Comment:
I think the original one ' Do you look forward to reducing your blockers not by management escalation but by collaboration?' means more like reduce the blockers by collaborating (doing it yourself) instead than waiting for other person to do it

Reply:
Any suggestion?

Reply:
I can't think of a good translation, but I'll leave this as an idea / hint:
`reducir esos momentos en que te encuentras atascado`

Reply:
Cant think of a good translation neither, this is the best i have
`¿Quieres eliminar tus bloqueos en lugar de esperar a que alguien mas lo haga?`